static/bidder-params/nextmillennium.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "NextMillennium Adapter Params",
+  "description": "A schema which validates params accepted by the NextMillennium adapter",
+  "type": "object",
+  "properties": {
+    "placement_id": {
+      "type": "string",
+      "description": "An id used to identify NextMillennium placement"
```

Comment:
Please add `"minLength": 1,` here as one of the `placement_id` fields so that the value can be validated as part of the JSON schema validation itself and you can remove the validation code above on L70 in `nextmillennium.go`